doc/HttpLuaModule.wiki

```diff
@@ -4698,6 +4699,56 @@ This feature was first introduced in the <code>v0.3.1rc22</code> release.
 
 See also [[#ngx.shared.DICT|ngx.shared.DICT]].
 
+== ngx.shared.DICT.cas ==
+'''syntax:''' ''success, err, forcible, current_value?, current_flags? = ngx.shared.DICT:cas(key, value, exptime, flags, old_value, old_flags?)''
+
+'''context:''' ''init_by_lua*, set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, body_filter_by_lua*, log_by_lua*, ngx.timer.*''
+
+Just like the [[#ngx.shared.DICT.replace|replace]] method, but only stores the key-value pair into the dictionary [[#ngx.shared.DICT|ngx.shared.DICT]] if the <code>old_value</code> argument and <code>old_flags</code> argument ''does'' match the value and flags in the dictionary [[#ngx.shared.DICT|ngx.shared.DICT]] (or one of them).
```

Comment:
Grammar: "do match" instead of "does match" since you have two things in the subject.